src/function/chatPostMessage.js

```diff
@@ -0,0 +1,72 @@
+import https from "https";
+import * as dotenv from "dotenv";
+
+dotenv.config();
+
+const TOKEN = process.env.LINE_ACCESS_TOKEN;
+
+// Request header
+const HEADERS = {
+	"Content-Type": "application/json",
+	Authorization: `Bearer ${TOKEN}`,
+};
+
+// Send four-day weather forecast information to user
+const chatPostMessage = function (replyToken, weatherForecastData, messageText) {
+	// Set four-day weather forecast information to message array
+	const message = [];
+
+	// Information retrieved from the weather API is every 3 hours for 5 days(40 data in total => 24h/3h*5d=40)
+	// Use one piece of information per day for four days
+	for (let i = 0; i < 32; i += 8) {
```

Comment:
maybe your reader understand where 32 and 8 come from but the code quality is bad now. is there any better way?

Reply:
I have changed this numbers to a constant to make it easier to understand.